php/admin/class-settings.php

```diff
@@ -84,6 +84,9 @@ public function render_page() {
 		$this->render_page_header();
 
 		switch ( $tab ) {
+			case 'blockcategories':
```

Comment:
I'd prefer not to introduce an new settings section if possible.

Reply:
Sounds good.